### ISSUE TITLE
Node js versions

### ISSUE THREAD

**Issue by joseph-norman** [winstonjs/winston-redis#14]
Hi, 

After installing `winston-redis` via npm I'm receiving the following warning:

```
npm WARN engine winston-redis@0.3.0: wanted: {"node":"0.8.x || 0.11.x"} (current: {"node":"0.10.33","npm":"2.1.11"})
```

I know it's not a big issue but perhaps this needs updating. Version 0.8 of node was released nearly 2 years ago. Same issue with npm - some major updates have been included.

**Comment by joseph-norman** [comment]
Update: just modified the `package.json` locally to include current versions and no immediate issues discovered.

**Comment by yads** [comment]
This should be resolved in v1.0.0
